source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -162,6 +162,16 @@ Then import the JavaScript file before the closing `</body>` tag of your HTML pa
 </body>
 ```
 
+Errors from components will be logged in the browser's console.
+
+For example, when:
+
+* GOV.UK Frontend is not supported in the current browser
+* Component templates have missing changes from our release notes
+* Component JavaScript configuration does not match our documentation
+
+You should check your application works without errors or some components will not work correctly.
```

Comment:
Thanks for adding this in 🎉

For context, it's taken from here (scroll down):
[**get-started**/#5-get-the-javascript-working](https://deploy-preview-333--govuk-frontend-docs-preview.netlify.app/get-started/#5-get-the-javascript-working)

>In your live application:
>
>- you must use `initAll` to initialise all components that use GOV.UK Frontend's JavaScript, or some components will not work correctly for disabled users who use assistive technologies